packages/cashier/src/Sass/crypto-transactions-history.scss

```diff
@@ -0,0 +1,148 @@
+.crypto-transactions-history {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+
+    &__header {
+        display: flex;
+        align-items: center;
+        margin-bottom: 1.6rem;
+        @include mobile {
+            display: block;
+            margin: 1.6rem;
+        }
+    }
+    &__back {
+        display: flex;
+        align-items: center;
+        .dc-text {
+            margin-left: 0.8rem;
+            width: max-content;
+        }
+    }
+    &__table {
+        display: flex;
+        flex-direction: column;
+        height: 100%;
+        @include mobile { 
+            padding: 1.6rem;
+        }
+        &-body {
+            flex: 1;
+            display: flex;
+            flex-direction: column;
+            @include desktop {
+                min-height: 72vh;
+                .data-list, .data-list__body-wrapper {
+                    flex: 1;
+                }
+            }
+        }
+        &-row {
+            padding: 1.6rem;
+            grid-template-columns: repeat(6, 1fr) 0.5fr;
+            border: 0;
+            .dc-table__cell:not(.crypto-transactions-history__table-confirm) {
+                @include desktop {
+                    margin-right: 1.6rem;
+                }
+            }
+            @include mobile {
+                display: flex;
+                flex-direction: column;
+                border-radius: 0.8rem;
+                background-color: var(--general-section-1);
+            }
+        }
+        &-cell {
+            margin-bottom: 0.8rem;
+        }
+        &-type {
+            text-transform: capitalize;
+            @include mobile {
+                margin-left: 1.2rem;
+                flex: 1;
+            }
+            .dc-text {
+                margin-left: 0.8rem;
+            }
+        }
+        &-amount {
+            @include mobile {
+                margin-bottom: 0.8rem;
+            }
+        }
+        &-hash {
+            a {
+                text-decoration-color: var(--brand-red-coral);
+            }
+            @include mobile {
+                margin-bottom: 0.8rem;
+            }
+        }
+        &-time {
+            @include mobile {
+                margin-bottom: 0.8rem;
+            }
+        }
+        &-status {
+            @include mobile {
+                display: flex;
+                align-items: center;
+            }
+            &-code {
+                width: 1rem;
+                height: 1rem;
+                margin-right: 0.8rem;
+                border-radius: 50%;
+                background-color: $color-yellow;
```

Comment:
Could you use the equivalent variable for `$color-yellow` from `theme.scss` instead.